src/CMakeLists.txt

```diff
@@ -223,8 +223,13 @@ else(INTEL_SSE4_1)
   message(STATUS "Skipping target ec_jerasure_sse4 & ec_shec_sse4: -msse4.1 not supported")
 endif(INTEL_SSE4_1)
 
+set(EXTRALIBS rt ${CMAKE_DL_LIBS} ${ATOMIC_OPS_LIBRARIES})
+if(LINUX)
+  set(LIB_RESOLV resolv)
+  set(LIB_UUID uuid)
```

Comment:
@tchaikov 
Feel free to drop it, as FreeBSD does not need it, hence the Linux block around it.
But until that moment, using uuid during linking uses an external lib, and cause breakage
Which I would need to install as a package first.